ui/tests/integration/utils/client-count-utils-test.js

```diff
@@ -7,595 +7,285 @@ import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
 import {
   filterVersionHistory,
-  flattenDataset,
   formatByMonths,
   formatByNamespace,
   homogenizeClientNaming,
   namespaceArrayToObject,
   sortMonthsByTimestamp,
 } from 'core/utils/client-count-utils';
 import { LICENSE_START } from 'vault/mirage/handlers/clients';
-import { parseAPITimestamp } from 'core/utils/date-formatters';
-import { addMonths, isAfter, isBefore } from 'date-fns';
-
-const MONTHS = [
-  {
-    timestamp: '2021-05-01T00:00:00Z',
-    counts: {
-      distinct_entities: 25,
-      non_entity_tokens: 25,
-      clients: 50,
-    },
-    namespaces: [
-      {
-        namespace_id: 'root',
-        namespace_path: '',
-        counts: {
-          distinct_entities: 13,
-          non_entity_tokens: 7,
-          clients: 20,
-        },
-        mounts: [
-          {
-            mount_path: 'auth/up2/',
-            counts: {
-              distinct_entities: 8,
-              non_entity_tokens: 0,
-              clients: 8,
-            },
-          },
-          {
-            mount_path: 'auth/up1/',
-            counts: {
-              distinct_entities: 0,
-              non_entity_tokens: 7,
-              clients: 7,
-            },
-          },
-        ],
-      },
-      {
-        namespace_id: 's07UR',
-        namespace_path: 'ns1/',
-        counts: {
-          distinct_entities: 5,
-          non_entity_tokens: 5,
-          clients: 10,
-        },
-        mounts: [
-          {
-            mount_path: 'auth/up1/',
-            counts: {
-              distinct_entities: 0,
-              non_entity_tokens: 5,
-              clients: 5,
-            },
-          },
-          {
-            mount_path: 'auth/up2/',
-            counts: {
-              distinct_entities: 5,
-              non_entity_tokens: 0,
-              clients: 5,
-            },
-          },
-        ],
-      },
-    ],
-    new_clients: {
+import { addMonths } from 'date-fns';
+
+const RESPONSE = {
```

Comment:
I assigned the response as a const here for reference, but also so we can confirm the numbers don't change. (Mirage generates new numbers each query). It's looooong but seems reasonable to have in this test as it helps discern how the response shape is changed to work for charts, etc.

Reply:
My only thought is to move this to a separate helper file (like we do with pki/values) so it doesn't take up so much room. But +1 to having consistent values!

Reply:
I agree! I'll do that in the next PR!